plutus-pab/README.md

```diff
@@ -0,0 +1,331 @@
+# Plutus Application Backend
+
+_PAB_ (Plutus Application Backend) is an Off-Chain application for managing the state of
+Plutus contract instances.
+
+## Table of Contents
+
+- [Building](#building)
+- [Running](#running)
+- [PAB Components](#pab-components)
+
+## Building
+
+_PAB_ is a Cabal based Haskell project. Production builds are done with Nix using [haskell.nix](https://github.com/input-output-hk/haskell.nix)
+
+### Cabal
+
+```
+$ cabal build
+```
+
+### Nix
+
+```
+$ nix-build ../default.nix -A plutus.haskell.packages.plutus-pab
+```
+
+## Running
+
+In order to use PAB several of its components need to be started. Furthermore the pab-client has
+to be started in order to access the web frontend. The required steps are described below.
+
+First we build the startup scripts:
+
+```
+$ nix-build ../default.nix plutus-pab.demo-scripts
+```
+
+Next we start all required servers and install several contracts:
+
+```
+$ ./result/bin/pab-start-all-servers
+```
+
+Now we start an additional node:
```

Comment:
This is not an additional node, it's an additional PAB connecting to the same node